tests/framework/matcher/phase.go

```diff
@@ -27,6 +27,16 @@ func HaveSucceeded() types.GomegaMatcher {
 	}
 }
 
+func PendingPopulation() types.GomegaMatcher {
+	return phaseMatcher{
+		expectedPhase: "PendingPopulation",
+	}
+}
+
+func WFFC() types.GomegaMatcher {
```

Comment:
Nit, but since we'll probably be using this matcher quite often I think `WaitForFirstConsumer()` is way prettier.

Reply:
ok done